docs/content/Installation/developer-setup.md

```diff
@@ -124,6 +123,20 @@ You would then run:
 
 To deploy the Operator on your Kubernetes cluster.  If your Kubernetes cluster is not local to your development host, you will need to specify a config file that will connect you to your Kubernetes cluster. See the Kubernetes documentation for details.
```

Comment:
lowercase the first " To" on this line since it's a continuation of the sentence above.